packages/next/build/index.ts

```diff
@@ -690,6 +690,18 @@ export default async function build(
       console.error(error)
       console.error()
 
+      // When using the web runtime, common Node.js native APIs are not available.
+      if (
+        hasConcurrentFeatures &&
+        error.indexOf("Module not found: Can't resolve 'fs'") > -1
```

Comment:
How about other native modules besides fs?

Reply:
For now, we only consider this case because almost all reports we get are caused by `fs`, and we certainly can't cover all native APIs.